features/tab-access-tabs.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Get or set TabStops
+  In order to customize tab stops
+  As a python-docx developer
+  I need a TabStops object with read/write tab_stop properties
+
+
+  @wip
+  Scenario Outline: Get tab_stop objects
+    Given a paragraph having <value> tab_stop elements
+     Then paragraph_format.tab_stops contain tab_stop objects
+      
+    Examples:  values for tab_stop elements 
+      | value |
+      | some  |
+      
+
+  @wip
+  Scenario Outline: Count tab stop objects
+    Given a paragraph having <value> tab_stop elements
+      Then paragraph_format.tab_stops has <lenval> length
+      
+    Examples:  length values for tab_stop elements
+      | value | lenval |
+      | no    | 0      |
+      | some  | 3      |
+      
```

Comment:
Structurally, there is no difference between 1 and 3 elements for tab_stops.  I can modify my sample document if you want, but it currently contains 2 paragraphs, one with no tab stops and one with 3 tab stops.  In my opinion, 2 tests should be adequate.